Makefile

```diff
@@ -320,8 +314,9 @@ kind-deploy: manifests
 
 .PHONY: kind-cluster
 kind-cluster: $(KIND) #EXHELP Standup a kind cluster.
+	env K8S_VERSION=v$(K8S_VERSION) KIND=$(KIND) GOBIN=$(GOBIN) hack/tools/validate_kindest_node.sh
```

Comment:
Didn't get to this yesterday, but would've preferred to keep this check separate from the cluster creation step, so we could include it in the verify target.